JeMPI_Apps/JeMPI_UI/src/components/import/DropZone.tsx

```diff
@@ -17,49 +18,65 @@ import { useSnackbar } from 'notistack'
 import { FC, useRef, useState } from 'react'
 import { FileRejection, useDropzone } from 'react-dropzone'
 import ApiClient from '../../services/ApiClient'
-import { FileObj, UploadStatus } from '../../types/FileUpload'
+import {
+  FileObj,
+  UploadStatus,
+  importQueriesType
+} from '../../types/FileUpload'
 import Button from '../shared/Button'
-import './Import.css'
 import UploadFileListItem from './UploadFileListItem'
 import { formatBytesSize, megabytesToBytes } from 'utils/formatters'
+import { useFormik } from 'formik'
 
 const MAX_UPLOAD_FILE_SIZE_IN_BYTES = megabytesToBytes(
   +(process.env.REACT_APP_MAX_UPLOAD_CSV_SIZE_IN_MEGABYTES ?? 128)
 )
+
 const DropZone: FC = () => {
+  const { enqueueSnackbar } = useSnackbar()
   const [fileObjs, setFilesObj] = useState<FileObj | undefined>()
   const abortControllerRef = useRef<AbortController>(new AbortController())
-  const { enqueueSnackbar } = useSnackbar()
+
+  const {
+    handleChange: handleImportFormChange,
+    handleSubmit,
+    values: FormValues,
+    setFieldValue
+  } = useFormik({
+    initialValues: {
+      reporting: false,
+      computing: 0,
+      leftMargin: 0.2,
+      rightMargin: 0.6,
+      threshold: 0.9,
+      windowSize: 0.1
+    },
+    onSubmit: () => {
+      if (fileObjs?.file) {
+        uploadFileMutation.mutate(fileObjs)
+      }
+    }
+  })
 
   const onDrop = (
     acceptedFiles: File[],
     fileRejections: FileRejection[]
   ): void => {
-    validate(acceptedFiles, fileRejections)
+    validate(fileRejections)
```

Comment:
Does `enqueSnackbar` in `validate` stop the process work flow? Wont `setFileObj` still get run even though there is an error?

Reply:
True, this issue was addressed, thank you @walisc.